src/AkeneoPimEnterpriseClient.php

```diff
@@ -108,7 +112,8 @@ public function __construct(
         ReferenceEntityMediaFileApiInterface $referenceEntityMediaFileApi,
         ReferenceEntityAttributeApiInterface $referenceEntityAttributeApi,
         ReferenceEntityAttributeOptionApiInterface $referenceEntityAttributeOptionApi,
-        ReferenceEntityApiInterface $referenceEntityApi
```

Comment:
We will see if we have time later and it's not the purpose of your PR. But it could be time to split these API by bounded contexts or something else?